packages/angular_devkit/build_angular/src/builders/browser-esbuild/schema.json

```diff
@@ -264,7 +264,7 @@
     },
     "vendorChunk": {
       "type": "boolean",
-      "description": "Generate a seperate bundle containing only vendor libraries. This option should only used for development.",
+      "description": "Generate a seperate bundle containing only vendor libraries. This option should only used for development to reduce the incremental compilation time.",
```

Comment:
This option isn't needed or used with esbuild but the addition is fine for now.  The option only exists here to prevent errors when trying out the new build system.